jsk_naoqi_robot/README.md

```diff
@@ -9,6 +9,7 @@ cd  catkin_ws
 wstool init src
 wstool merge -t src https://raw.githubusercontent.com/jsk-ros-pkg/jsk_robot/master/jsk_naoqi_robot/pepper.rosinstall
 wstool update -t src
+;; Be sure that ROS_DISTRO is set (echo $ROS_DISTRO). If not, execute "source /opt/ros/indigo/setup.bash"
 rosdep install -y -r --from-paths src --ignore-src
```

Comment:
@kochigami updating this line to
` rosdep install -y -r --from-paths src --ignore-src --rosdistro=XXXX`
is better than just documenting.
Also `;;` is for lisp not a shell.